### PR TITLE
Product listing page

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
Product listing page

### CODE REVIEW

**Comment by som749898** [comment]
looks good to me

File: ProductListing/productListing.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
+.filter {
+    height: 100vh;
+    padding: 1rem;
+    width: 300px;
```
Comment (ranitasahaa1230) [inline ProductListing/productListing.css:4]: You can use rems over here

File: ProductListing/productListing.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <title>TheSportsAdda | Product Listing</title>
```
Comment (ranitasahaa1230) [inline ProductListing/productListing.html]: You can add desc in your meta tag like author name, page content etc. Please delete the  first title

File: ProductListing/productListing.html [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+    <link href="https://ozone-ui.netlify.app/Docs/doc.css" rel="stylesheet" type="text/css">
+    <link href="https://ozone-ui.netlify.app/Components/Button/button.css" rel="stylesheet" type="text/css">
+    <link href="../CSS/nav.css" rel="stylesheet" type="text/css">
+    <link href="../CSS/utility.css" rel="stylesheet" type="text/css">
+    <link href="../ProductListing/productListing.css" rel="stylesheet" type="text/css">
+    <link href="../Wishlist/wishlist.css" rel="stylesheet" type="text/css">
```
Comment (ranitasahaa1230) [inline ProductListing/productListing.html]: You can put all these links in a CSS folder and just add that URL to your main style.css file

File: ProductListing/productListing.html [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+
+            <h4 class="margin-top">Sort By</h4>
+            <div>
+                <input type="radio" name="sort-by" />
+            <label>Price - Low to High</label><br>
+            <input type="radio" name="sort-by" />
+            <label>Price - High to Low</label><br>
+
+            </div>
+
```
Comment (ranitasahaa1230) [inline ProductListing/productListing.html]: Remove all these extra spaces

File: CSS/utility.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+}
+.text-strike {
+    text-decoration: line-through;
+}
+.flex-center {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+/* .flex-column {
```
Comment (JaanviNL) [inline CSS/utility.css]: remove the commented code

**Review by ranitasahaa1230 — APPROVED** [review]
Suggested a few changes, Rest looks good

**Review by JaanviNL — APPROVED** [review]
Rest all Seems good

File: ProductListing/productListing.css (hunk 1 above)
Comment (govardhanreddyc79) [reply ProductListing/productListing.css:4]: okay, Thanks

File: ProductListing/productListing.html (hunk 2 above)
Comment (govardhanreddyc79) [reply ProductListing/productListing.html]: sure.

File: ProductListing/productListing.html (hunk 3 above)
Comment (govardhanreddyc79) [reply ProductListing/productListing.html]: okay

File: ProductListing/productListing.html (hunk 4 above)
Comment (govardhanreddyc79) [reply ProductListing/productListing.html]: sure,Thanks

File: CSS/utility.css (hunk 5 above)
Comment (govardhanreddyc79) [reply CSS/utility.css]: okay

**Pull request merged**